set_env_db_template.sh

```diff
@@ -0,0 +1,10 @@
+export DB_API_PORT=27017 #Port on the MongoDB server
+export DB_SERVER='cta-simpipe-protodb.zeuthen.desy.de' # MongoDB server
+export DB_API_USER=YOUR USERNAME # username for MongoDB: ask the responsible person
```

Comment:
Suggest to have `YOUR_USERNAME`, as spaces are not allowed. Same for the password.